src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -0,0 +1,76 @@
+package org.fundacionjala.sfdc.util;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * CommonActions.java
+ * Class with common methods that many classes can use.
+ */
+public final class CommonActions {
+
+    /**
+     * Private constructor because it is a util class.
+     */
+    private CommonActions() {
+
+    }
+
+    /**
+     * Method to get any element but wait until it is visible.
+     * @param element to wait for.
+     * @return the element.
+     */
+    public static WebElement getElement(final WebElement element) {
+        DriverManager.getInstance().getWaiter().until(ExpectedConditions.visibilityOf(element));
+        return element;
+    }
+
+    /**
+     * Method to get any element but wait until it is visible.
+     * @param element to wait for.
+     * @return the element.
+     */
+    public static By getElement(final By element) {
+        DriverManager.getInstance().getWaiter().until(ExpectedConditions.visibilityOfElementLocated(element));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)